psi-sample/enclave/src/main/java/com/r3/conclave/sample/enclave/PSIEnclave.java

```diff
@@ -24,7 +24,7 @@
 public class PSIEnclave extends Enclave {
 
     private static final String MERCHANT = "MERCHANT";
-    private static final String SERVICE_PROVIDER = "SERVICE-PROVIDER";
+    private static final String SERVICE_PROVIDER = "SERVICE_PROVIDER";
```

Comment:
To showcase good Conclave app design, you should move the `Role` enum to the `common` module and then use it in the Enclave as well, rather than having these constants.

Reply:
done